touchforms/formplayer/api.py

```diff
@@ -320,8 +321,9 @@ def post_data(data, auth=None, content_type="application/json"):
     if auth:
         d['hq_auth'] = auth.to_dict()
     # just default to old server for now
-
-    if toggles.SMS_USE_FORMPLAYER.enabled():
+    domain = d.get("domain")
```

Comment:
where does domain come from? i don't think it's getting passed on every request. when i add these lines here

    if not domain:
            raise RuntimeError()

and run the tests, it raises the RuntimeError